assets/src/settings/index.js

```diff
@@ -56,50 +66,51 @@ class Settings extends Component {
 		} = this.state;
 
 		return (
-			<div className="ncc-page">
-				<h1>{ __( 'Content Conversion Settings' ) }</h1>
-				<br />
-
-				<p>
-					{ __(
-						'Adding content to the queue, enables it to be converted to Gutenberg Blocks. The queue is also a backup point for possible reverting.'
-					) }
-				</p>
-				<p>
-					{ __(
-						'Existing HTML content is first selected by type, and added to a conversion queue. Queued content is then converted.'
-					) }
-				</p>
-				<h3>{ __( 'Specify content type' ) }</h3>
-				<ul>
-					<li>
-						content types:{' '}
-						<input type="text" disabled={ true } value={ conversionContentTypesCsv } />
-					</li>
-					<li>
-						content statuses:{' '}
-						<input type="text" disabled={ true } value={ conversionContentStatusesCsv } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Conversion params' ) }</h3>
-				<ul>
-					<li>
-						conversion batch size:{' '}
-						<input type="text" disabled={ true } value={ conversionBatchSize } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Queued stats' ) }</h3>
-				<ul>
-					<li>
-						<b>
-							<u>{ queuedEntries }</u>
-						</b>{' '}
-						posts are currently selected and queued for conversion
-					</li>
-				</ul>
-			</div>
+			<Fragment>
+				<div className="newspack-logo-wrapper">
+					<NewspackLogo />
+				</div>
+				<Grid>
+					<FormattedHeader
+						headerIcon={ <SettingsIcon /> }
+						headerText={ __( 'Content conversion settings' ) }
+						subHeaderText={ __( 'Adding content to the queue to convert it to Gutenberg Blocks.' ) }
+					/>
+					<Card>
+						<p>
+							{ __(
+								'Existing HTML content is first selected by type, and added to a conversion queue. Queued content is then converted. The queue is also a backup point for possible reverting.'
+							) }
+						</p>
+						<hr />
+						<h2>{ __( 'Specify content type' ) }</h2>
+						<TextControl
+							label={ __( 'Content types' ) }
+							disabled={ true }
+							value={ conversionContentTypesCsv }
+						/>
+						<TextControl
+							label={ __( 'Content statuses' ) }
+							disabled={ true }
+							value={ conversionContentStatusesCsv }
+						/>
+						<hr />
+						<h2>{ __( 'Conversion params' ) }</h2>
```

Comment:
Looking at these two, they can be totally removed:
- Conversion params -- Conversion batch size -- this is an internal param, definitely better not to let the user change this
- Queued stats -- number of entries to be converted -- this is displayed on the "Run Conversion" page, no need for double here.

I'm opening [an Issue for myself here to remove this](https://github.com/Automattic/newspack-content-converter/issues/42), no need to do it here. I'm sharing for two reasons -- please notice if this will change the overall UI, and we can talk about more, and also if you think I'm wrong please share your comments.

Reply:
I will leave this one for you then 😊

Reply:
I actually took care of this in 5c6adb4.

I also simplified the text on the page.